src/core/mount.c

```diff
@@ -1759,14 +1762,51 @@ static void mount_enumerate(Manager *m) {
         mount_shutdown(m);
 }
 
+static void mount_process_proc_self_mountinfo(Manager *m);
+
 static int mount_dispatch_io(sd_event_source *source, int fd, uint32_t revents, void *userdata) {
-        _cleanup_set_free_ Set *around = NULL, *gone = NULL;
         Manager *m = userdata;
-        const char *what;
-        Iterator i;
-        Unit *u;
         int r;
 
+        if (now(CLOCK_MONOTONIC) < usec_add(m->mount_last_read_usec,
+                                            m->mount_last_duration_usec * 10)) {
```

Comment:
not that it matters, but given that we do the same usec_add() invocation again a few lines down, maybe just store the result in a variable here and reuse it?